libc-test/build.rs

```diff
@@ -4480,9 +4480,9 @@ fn test_linux(target: &str) {
     });
 
     cfg.skip_roundtrip(move |s| match s {
-        // FIXME:
+        // FIXME(ctest):
```

Comment:
Same as the other PR, a couple instances of `FIXME(ctest)` that should be something else. Rest looks good!